.github/workflows/build.yml

```diff
@@ -23,19 +27,14 @@ jobs:
     runs-on: ubuntu-latest
     timeout-minutes: 10
     steps:
-      - name: Cancel Previous Runs
-        uses: styfle/cancel-workflow-action@0.11.0
-        with:
-          access_token: ${{ github.token }}
-
       - name: Checkout
         uses: actions/checkout@v3
```

Comment:
v4?

Reply:
I'm letting dependabot do the updates to check whether it's working properly or not o/